lib/tapioca/cli.rb

```diff
@@ -21,10 +21,18 @@ class Cli < Thor
       desc: "Verbose output for debugging purposes",
       default: false
 
-    desc "init", "initializes folder structure"
-    option :postrequire, type: :string, default: DEFAULT_POSTREQUIRE_FILE
+    desc "init", "get project ready for type checking"
     def init
-      command = Commands::Init.new(
+      invoke(:configure)
+      invoke(:annotations)
+      invoke(:gem)
+      invoke(:todo)
```

Comment:
Yeah this makes me slightly nervous too. I think we should decide what we value most with this change. If we value having tapioca "just work" when a user sets it up, then running `todo` makes sense. If we value avoiding generating potentially incorrect types, then it makes sense to take it out.

Maybe we can also include a note in the "What's next" message (see comment below this one) reminding the user to look through the types generated in `todo.rbi` to check for correctness?

Reply:
It just occurred to me - this is even more of a problem if we're not compiling DSLs, right? It is likely that todo will generate signatures for constants that would be generated via DSLs, which could create a bad experience.

Reply:
> reminding the user to look through the types generated in todo.rbi to check for correctness?

Yeah I think we should make it clear in the output that this file should be reviewed carefully as it would hide potential real error.

> todo will generate signatures for constants that would be generated via DSLs

Not sure what you mean here as `todo` only generates `module X; end` for each undefined constant X. So no real signature? Having useless constants in this file after generating DSL should be caught by `check-shims`, maybe we should mention something about this command in the what's next?

Reply:
Yes, exactly. I'm not sure what the best approach is. Running `todo` certainly helps the adoption, since it just makes type checking pass, but it doesn't provide very accurate typing information.

Not a blocker for the PR though, just a thought on how we'd like to have this setup.

Reply:
Oh I think I imagined worse consequences for having RBI definitions for constants that don't exist in the app. If it won't practically cause any problems then I'm less worried about it!

Reply:
It still is a problem as it may be hiding a non-defining constant that should actually be reported as an error. This should be explained properly the message printed after `tapioca init`. Or maybe the message for `tapioca todo` should be enhanced first.